.circleci/config.yml

```diff
@@ -149,6 +171,43 @@ jobs:
       - run:
           name: Test Logging.
           <<: *anchor_test_ruby
+  test-php-7.0:
```

Comment:
Why 7.0? If we had to pick only one version, then probably 7.2 is best.

Reply:
Sure!